init.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
This is great!

I've seen other projects use `composer create-project` ([docs](https://getcomposer.org/doc/03-cli.md#create-project)) a standard way to:

1. clone the template repository and reset the git history,
2. run `composer install`, and
3. run the scaffolding scripts automatically through the `post-create-project-cmd` [hook](https://getcomposer.org/doc/articles/scripts.md#command-events).

Switching to a PHP based scaffolding script would also allow us to decouple the provisioning script from Bash and mosty *nix systems. Ideally, all scripts would be either PHP or JS which have reliable binaries on all platforms.

See this [scaffolding controller](https://github.com/xwp/block-extend/blob/cc49ce9e8f3f8b45df366348d034740fc68feb3e/scripts/composer/PluginScaffolder.php) for example.